Makefile

```diff
@@ -67,7 +67,7 @@ TEST = $(TEST_CMD) $(TEST_PKGS)
 DOCKER_BUILD ?= $(DOCKER) run -it --rm -v $(DIR):$(BUILDMNT) -w $(BUILDMNT) $(BUILD_IMAGE) /bin/sh -c
 
 # TODO (irfanurrehman): can add local compile, and auto-generate targets also if needed
-.PHONY: all container push clean hyperfed controller kubefedctl test local-test vet fmt build bindir generate webhook e2e deploy.kind
+.PHONY: all container push clean hyperfed controller kubefedctl test local-test vet build bindir generate webhook e2e deploy.kind
```

Comment:
I think it will be useful to have an option like `make lint` for a developer to run before submitting a PR. I mean one can run it explicitly also, but having it as part of `make`, makes it intuitive. WDYT?

Reply:
Sure, can do. I was going to add [pre-commit](https://pre-commit.com) config to make it easier to run but can absolutely add `lint` target to makefile.

Reply:
Added lint target.